aggregator.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"encoding/base64"
```

Comment:
Is there a reason all indentations are tabs? Not to start a war or something but it is kinda weird...

Reply:
both ways are acceptible. 
And the good thing about go - there is no place for holy war, `go fmt` resolves all doubts